documentation/architecture/desktop_application/data-model.md

```diff
@@ -0,0 +1,80 @@
+# Desktop application
+
+This document presents the data model used in the desktop application.
+
+There are 3 main activities available to a user when using the application:
+
+1. Enroll / Reset / Quit
+2. Share a local service / Invite a user to a shared service
+3. Accept an invitation / Connect to a remote service
+
+The following sections show the main data types supporting the scenarios above
+
+# Enrollment
+
+![enroll](./diagrams/enroll.png)
+
+When a user enrolls with the application:
+
+ 1. a space is either created or retrieved for them (if they previously enrolled with the same email address)
+ 2. a project is either created or retrieved for them (if they previously enrolled with the same email address)
+ 3. a relay is created in order to support the routing of all the shared services
```

Comment:
Can we link to the dedicated docs page on the word `relay`, the first time it appears in this document? Same with secure channel, probably?

Reply:
I have added two links.